examples/aws-credentials-secret.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Secret
+metadata:
+    name: aws-access-keys
+type: Opaque
+data:
+    aws_access_key_id: ba3b9f8f-c9b7-4b5b-b8b7-f8f9f8f8f8f8
+    aws_secret_access_key: 87ab23c5ef5190a
```

Comment:
maybe better mark that these are not real credentials and use some pattern or [Hexspeak](https://en.wikipedia.org/wiki/Hexspeak)? It took me a second to realize that this was not a leaking credentials case...
```
    aws_access_key_id: deadbeef12345678
    aws_secret_access_key: deadbeef12345678deadbeef12345678
```

Reply:
Great idea